### PR TITLE
Adds link to Maven repo and updates plugins list with Geospatial

### LINKED ISSUE #838: [BUG] Add Maven coordinates to list of available plugins
### What is the bug?

Someone at the community meeting brought this up today - the idea is that we can use the plugin install script with an official plugin name to install a plugin. But -- when visiting https://opensearch.org/docs/1.3/opensearch/install/plugins/#install-a-plugin , we state that plugins can be installed via "Maven Coordinates to a plugin zip file" as well as "A URL to a plugin zip file"

As we offer a list of official plugin names, but not a list of Maven Coordinates or URLs. They use Maven, and found themselves without guidance at all on where to get said plugins. 

### How can one reproduce the bug?

Visit https://opensearch.org/docs/1.3/opensearch/install/plugins/#install-a-plugin

### What is the expected behavior?

A reference of Maven Coordinates for the plugins along with the list of official plugin names.

### LINKED ISSUE #998: [DOC] Add Geospatial to plugins list
**What do you want to do?**
 
Add OpenSearch Geospatial to https://opensearch.org/docs/latest/opensearch/install/plugins/

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
 
The plugin was added with 2.2.0 but there was no docs ticket.

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.

n/a

### PR DESCRIPTION
**Pull request by JeffH-AWS**
Signed-off-by: JeffH-AWS <jeffhuss@amazon.com>

### Description
Added link to the official Maven repo, and added Geospatial to the list of official plugins.

### Issues Resolved
Fixes #998 
Fixes #838 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by lizsnyder — APPROVED** [review]

**Review by Naarcha-AWS — APPROVED** [review]

**Review by cwillum — APPROVED** [review]
LGTM

File: _opensearch/install/plugins.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ If you are running OpenSearch in a Docker container, plugins must be installed,
 The install command takes a plugin id, which may be any of the following:
 
 - An official OpenSearch plugin name
-- Maven coordinates to a plugin zip file
+- [Maven coordinates](https://mvnrepository.com/artifact/org.opensearch.plugin) to a plugin zip file
```
Comment (rursprung) [inline _opensearch/install/plugins.md:28]: @JeffH-AWS:

i just saw these docs for the first time.

please note that this is _not_ correct (unless somebody just added that feature? but i haven't seen anything in these regards)! `opensearch-plugin install` does *not* accept maven coordinates. you cannot e.g. write `opensearch-plugin install org.opensearch.plugin:opensearch-security:2.2.0.0`! (though you could of course use the normal HTTP URL of that maven artefact, but then it's irrelevant that it's on maven)

the usage for maven here is in combination with automated build systems which need to fetch the artefacts (often through proxies in company environments), cache them and then later feed them to a build (e.g. docker build) which takes them as normal files and calls the install command with a `file:///` path

Comment (rursprung) [reply _opensearch/install/plugins.md:28]: [the blog post](https://opensearch.org/blog/technical-post/2022/08/opensearch-plugin-zips-now-in-maven-repo/) might contain further information relevant for you as it describes how the maven artefacts can be used

Comment (JeffH-AWS) [reply _opensearch/install/plugins.md:28]: Thanks for calling that out Ralph!  I was not aware that the original documentation was also inaccurate - nobody has raised an issue on that.  Would you mind opening a new issue?  You can even assign it to me, if that's something you're able to do.

Comment (rursprung) [reply _opensearch/install/plugins.md:28]: np. i've raised #1007. i don't have the rights to assign it to you, though.

**Pull request merged**